legal-api/report-templates/template-parts/alteration-notice/nameTranslation.html

```diff
@@ -0,0 +1,20 @@
+<div class="no-page-break">
```

Comment:
nameTranslations are in ```report-templates/template-parts/notice-of-articles/benefitCompanyStmt.html``` and should be common

Reply:
This has changed to comply with the review.